### PR TITLE
Add how-to guide for PostgreSQL installation via Helm

### PR DESCRIPTION
**Pull request by olimination**
This how-to guide is the docs result of the question/discussion here: https://github.com/appuio/appuio-cloud-community/discussions/27

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

**Comment by olimination** [comment]
Hi @simu ,
I have created a how-to according to my question and your response here: https://github.com/appuio/appuio-cloud-community/discussions/27

Maybe you would like to review.

Unfortunately I was not able to set the "how-to" label...maybe I don't have enough permissions to do it or I just don't know how :)?

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+== Step 2: Create an adapted values.yaml file
+
+We use the Bitnami provided PostgreSQL helm chart documented here: https://github.com/bitnami/charts/tree/master/bitnami/postgresql
+
+For the usage in {product} it is needed to override some chart parameters, for example the `securityContext` parameter or
+the resource limits.
+
+If users don't explicitly configure limits, {product} platform injects limits of cpu: 200m and memory: 200Mi. 
+(see https://docs.appuio.cloud/user/references/default-quota.html#_resource_limits_and_defaults).
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:28]:
```suggestion
See the reference docs of {product} for more details on xref:references/default-quota.adoc#_resource_limits_and_defaults[default resource limits].
```

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+See also the `values.yaml` file of the PostgreSQL Helm chart for more details: 
+https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:31]:
```suggestion
See also the https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml[`values.yaml` file of the PostgreSQL Helm chart] for more details on configurable values.
```

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+Create some custom `postgres-values.yaml` (filename does not matter) file to override the needed chart parameters.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:33]:
```suggestion
Create a custom `postgres-values.yaml` (filename does not matter) file to override the needed chart parameters.
```

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+See here a full working example of this custom values file:
+
+[source,yaml]
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:37]:
```suggestion
Below is a full working example for deploying the Bitnami PostgreSQL chart on {product}.

.postgres-values.yaml
[source,yaml]
```

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+shmVolume:
+  chmod:
+    enabled: false
+
+containerSecurityContext:
+  enabled: false
+----
+<1> Set your database user name
+<2> Set some valid PostgreSQL image tag
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:70]:
```suggestion
<2> Set a valid PostgreSQL image tag
```

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+By default the `storageClass` is empty and the {product} platform uses `ssd` as the default storage class.
+
+See also the following link for more details about the available storage classes:
+https://docs.appuio.cloud/user/explanation/storage-classes.html#_cloudscale_ch
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:75]:
```suggestion
See the xref:explanation/storage-classes.adoc[documentation of available storage classes] on {product} for more details.
```

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+== Step 3: Install the Helm chart with the custom values.yaml file
+
+Install the PostgreSQL Helm chart with the following command:
+
+[source,shell]
+----
+helm install -f postgresql-values.yaml --set global.postgresql.postgresqlPassword=<DB_PW> <MY_HELM_INSTANCE_NAME> bitnami/postgresql
+----
+
+Just replace the `<DB_PW>` with your database user password.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:87]:
```suggestion
Replace the `<DB_PW>` with your database user password.
```

File: docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+Replace `<MY_HELM_INSTANCE_NAME>` with some descriptive name for your Helm installation, for example `service-xy-db`.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/install-postgres-db-helm.adoc:89]:
```suggestion
Replace `<MY_HELM_INSTANCE_NAME>` with a descriptive name for your Helm installation, for example `service-xy-db`.
```

**Review by simu — APPROVED** [review]
Thanks for the contribution :tada: 

I added some inline comments (mostly language nitpicks and small wording adjustments). It would be great if you could add them to the PR.

**Comment by olimination** [comment]
Thanks @simu for the review.

I have added your inputs in the following commit: https://github.com/appuio/appuio-cloud-docs/pull/83/commits/396b63c86a2293f6248417296a12db2e41f9c6a7

**Comment by tobru** [comment]
Thanks @olimination for the contribution :rocket: You can see the result here: https://docs.appuio.cloud/user/how-to/install-postgres-db-helm.html

**Pull request merged**